en/docs/references/alerting.md

```diff
@@ -73,3 +73,64 @@ In addition, you can observe the following:
 - The time range for which the error log applies (marked **3** in the image). However, you can scroll to view more logs that occurred during the time range selected for debugging purposes. This time range is selected via the drop-down field marked **5** in the image. You can update this time range as explained under [Select a custom time range](#select-a-custom-time-range).
 
 - The status and the latency of the request (marked **4** in the image).
+
+### Anomaly alert 
+
+This alert is triggered when the current latency shows a significantly sudden upward shift compared to the average latency computed in the last 5 minutes of a resource function. Latency spikes and upward latency shifts are considered high latency. 
+
+Anomaly alerting is disabled by default. An organization administrator needs to enable it. 
+
+Choreo alerts a user of this anomaly via email. This email includes the component name, the organization the component belongs to, the environment it occurred, version, and anomaly detection information. The email has a link that takes a user to the components' latency graph in the observability view pane. A user can view the latency graph by clicking on the link **Check in Portal** in the email and and thereby [perform an RCA](https://wso2.com/choreo/docs/observability/root-cause-analysis/) for the anomaly. 
```

Comment:
can't we say this includes details to identify the relevant component and the resource function anomaly time stamp etc?

Reply:
changed.